riscv-executor/src/lib.rs

```diff
@@ -272,13 +374,18 @@ mod builder {
     use powdr_number::FieldElement;
 
     use crate::{
-        Elem, ExecMode, ExecutionTrace, MemOperation, MemOperationKind, MemoryState, RegWrite,
-        RegisterMemory, RegisterMemoryState, PC_INITIAL_VAL,
+        BinaryMachine, Elem, ExecMode, Execution, ExecutionTrace, MemOperation, MemOperationKind,
+        MemoryMachine, MemoryState, RegWrite, RegisterMemory, ShiftMachine, SplitGlMachine,
+        Submachine, SubmachineBoxed, PC_INITIAL_VAL,
     };
 
+    // TODO: where do we get this from? I don't think we want to depend on the linker crate just for this
```

Comment:
It should be in the parsed asm somewhere. Both min and max are defined explicitly in the current impl of the riscv gl machine.